Microsoft.Dynamics365.UIAutomation.Api/Pages/Entity.cs

```diff
@@ -986,6 +986,43 @@ public BrowserCommandResult<string> GetHeaderValue(string field)
             });
         }
 
+        /// <summary>
+        /// Get all the available header option values from a dropdown
+        /// </summary>
+        /// <param name="option">The option you want to get values.</param>
+        /// <returns>The list of options</returns>
+        /// <example>xrmBrowser.Entity.GetHeaderOptionValues("status");</example>
```

Comment:
Could you update this example to reflect usage of the OptionSet control type?

Reply:
updated thank you